hooks/post/30-upgrade.rb

```diff
@@ -1,4 +1,6 @@
 if app_value(:upgrade)
+  noop = app_value(:noop) ? ' (noop)' : ''
```

Comment:
https://github.com/Katello/katello-installer/pull/236/files#diff-1453f17351c9fefb73f41a2dd0837b82R5 references it, and it was before it was defined.  Do you want a better commit message maybe?